docs/services/topology-contacts-data.md

```diff
@@ -0,0 +1,117 @@
+Topology and Contacts Data
+==========================
+
+This document contains information about the data provided by <https://topology.opensciencegrid.org>.
```

Comment:
Let's try and warn people that this doc is for internal use in case they accidentally stumble upon it:

> This is internal documentation intended for OSG Operations staff. It contains information about...

Or something like that